Java/JDI/jdi-uitest-web/src/main/java/com/epam/jdi/uitests/web/selenium/elements/complex/RadioButtons.java

```diff
@@ -39,4 +52,22 @@ public RadioButtons(By optionsNamesLocatorTemplate, By allOptionsNamesLocator) {
         super(optionsNamesLocatorTemplate, allOptionsNamesLocator);
     }
 
+
+    public static void setUp(BaseElement el, Field field) {
+        if (!fieldHasAnnotation(field, JRadioButtons.class, IRadioButtons.class))
+            return;
```

Comment:
Please surround with curly braces. And try to avoid if-else (and similar code structures) statements without using curly.